js/src/lib/interfaces/Libraries.ts

```diff
@@ -73,7 +73,7 @@ export interface LibraryUiElement {
 
 function nameWithoutNamespace(modelId: string): string {
 	const splitted = modelId.split("/");
-	return splitted.length === 1 ? splitted[0] : splitted[1];
+	return splitted[1] ?? splitted[0] ?? "";
```

Comment:
```suggestion
	return splitted[1] ?? splitted[0];
```
I think that's enough, we'll necessarily have something in `splitted[0]` (at the minimum it will be `""`)

Reply:
yes but TypeScript doesn't know  about it :( 

<img width="522" alt="Screen Shot 2023-08-25 at 2 12 31 PM" src="https://github.com/huggingface/hub-docs/assets/8323351/e6be6480-2a69-4de6-8f7b-330c675a2a53">

Reply:
Actually now that we're using hub-docs as a dependency, I don't think typescript is checking the files on our side. I don't mind removing these sort of changes.

I'll argue that as a library shared between different projects, and involving numerous contributors, maybe it would make sense to move toward stricter type checking options, as an easy way to help prevent unforeseen bugs and crashes.

Reply:
> we have a stricter tsconfig on the Endpoints side, with notably these params:
> 
> ```
>     "noUncheckedIndexedAccess": true,
>     "noImplicitAny": true,
> ```

i think we should align on the type-checking settings of moon-landing. So if you feel strongly about that, you should open a PR to change this config on moon-landing first

Reply:
> Actually now that we're using hub-docs as a dependency, I don't think typescript is checking the files on our side. I don't mind removing these sort of changes from this PR.

Yes please then, `noUncheckedIndexedAccess` is a bit hardcore honestly (and wrong, in this case)

Reply:
@Pierrci it's not totally wrong because there are instances where the result of String.split() could be an empty array, but it's true that's not the case here :)